packages/expo-blur/android/src/main/java/expo/modules/blur/ExpoBlurView.kt

```diff
@@ -0,0 +1,53 @@
+package expo.modules.blur
+
+import android.content.Context
+import android.os.Build
+import android.view.ViewGroup
+import eightbitlab.com.blurview.BlurView
+import eightbitlab.com.blurview.RenderEffectBlur
+import eightbitlab.com.blurview.RenderScriptBlur
+import expo.modules.kotlin.AppContext
+import expo.modules.kotlin.views.ExpoView
+
+class ExpoBlurView(context: Context, appContext: AppContext) : ExpoView(context, appContext) {
+  private var blurReduction = 3.5f
+  private var blurRadius = 3f
+
+  private val blurView = BlurView(context).also {
+    it.layoutParams = LayoutParams(LayoutParams.MATCH_PARENT, LayoutParams.MATCH_PARENT)
+
+    val decorView = appContext.currentActivity?.window?.decorView
```

Comment:
```suggestion
    val decorView = (appContext.currentActivity ?: throw Exceptions.MissingActivity()).window.decorView
```
It would be better to provide a meaningful exception here. The code will crash anyway with NullPointerException later.